src/components/AddItem/AddItem.js

```diff
@@ -84,6 +97,16 @@ const AddItem = (props) => {
           />
         </label>
         <br />
+        <label htmlFor="addItem">
+          Last Purchase Date:
```

Comment:
Cool idea to add an optional last purchase date field here! We might want to label it as optional as well. So users don't have to think too hard if they can't remember the last purchase date. What do you think?